src/daemon/common_daemon_test.h

```diff
@@ -21,6 +22,7 @@ struct CommonDaemonTest : public ::testing::Test {
   using RunCallback = Fn<void(base::TestProcess*)>;
 
   void SetUp() override {
+    DumpStatisticValues();
```

Comment:
GTEST runs tests in same process if you don't ask it to spawn separate process for every new test. It means that several tests share same global objects like PerfService. To make results of one test not mess with results of other one it would be nice to clear results before every test run.